src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -78,15 +83,30 @@ public UpdatePullRequestCommentsSteps(GitHubService gitHubService, GitHubPropert
         this.adoControllerSpy = Mockito.spy(adoController);
         this.flowProperties = flowProperties;
         this.cxProperties = cxProperties;
-        initGitHubProperties();
+        this.scaProperties = scaProperties;
+
+    }
+
+    private void initSca() {
+        scaProperties.setAppUrl("https://sca.scacheckmarx.com");
+        scaProperties.setApiUrl("https://api.scacheckmarx.com");
+        scaProperties.setAccessControlUrl("https://platform.checkmarx.net");
```

Comment:
Why not using them from the ScaCommonSteps -> initSCAConfig()?

Reply:
I told you on the phone - I will have to inject an unnecessary dependency in order to extend it, so I think it is better this way.
Anyway, we can think of making this hierarchy more flexible somehow, to make it easier to use.
What do you think ?